rust/src/lib.rs

```diff
@@ -380,6 +381,75 @@ pub fn version() -> string::BnString {
     unsafe { string::BnString::from_raw(binaryninjacore_sys::BNGetVersionString()) }
 }
 
+pub fn build_id() -> u32 {
+    unsafe { binaryninjacore_sys::BNGetBuildId() }
+}
+
+#[derive(Clone, PartialEq, Eq, Hash)]
+pub struct VersionInfo {
+    pub major: u32,
+    pub minor: u32,
+    pub build: u32,
+    pub channel: string::BnString,
+}
+
+pub fn version_info() -> VersionInfo {
+    let info_raw = unsafe { binaryninjacore_sys::BNGetVersionInfo() };
+    VersionInfo {
+        major: info_raw.major,
+        minor: info_raw.minor,
+        build: info_raw.build,
+        channel: unsafe { string::BnString::from_raw(info_raw.channel) },
```

Comment:
This will crash on local builds of binary ninja due to a bug for any other reviewers.

Reply:
Fix is in latest dev for the crash so no cause for alarm anymore :)